src/main/resources/assets/js/widgets/layers/LayersWidgetItemView.ts

```diff
@@ -8,6 +7,14 @@ import {LayerContent} from './LayerContent';
 import {DefaultErrorHandler} from 'lib-admin-ui/DefaultErrorHandler';
 import {DivEl} from 'lib-admin-ui/dom/DivEl';
 import {LayersContentTreeDialog} from './dialog/LayersContentTreeDialog';
+import {ProjectCreatedEvent} from 'lib-contentstudio/app/settings/event/ProjectCreatedEvent';
+import {ProjectUpdatedEvent} from 'lib-contentstudio/app/settings/event/ProjectUpdatedEvent';
+import {ProjectDeletedEvent} from 'lib-contentstudio/app/settings/event/ProjectDeletedEvent';
+import {Store} from 'lib-admin-ui/store/Store';
+import {ContentServerEventsHandler} from 'lib-contentstudio/app/event/ContentServerEventsHandler';
+import {ContentServerChangeItem} from 'lib-contentstudio/app/event/ContentServerChangeItem';
+
+export const LAYERS_WIDGET_ITEM_VIEW = 'LayersWidgetItemView';
```

Comment:
`LayersWidgetItemView.name` should give you class name, no need to create a const.